docs/pages/index.tsx

```diff
@@ -211,18 +210,26 @@ const Home = () => {
         </Description>
         <CellContainer>
           <Row>
-            <APIGridCell title="Maps" link="/versions/latest/sdk/map-view" icon={<APIMapsIcon />} />
+            <APIGridCell
+              title="Maps"
+              link="/versions/latest/sdk/map-view"
+              icon={<Map01DuotoneIcon className={mergeClasses('h-16 w-16')} />}
```

Comment:
```suggestion
              icon={<Map01DuotoneIcon className="h-16 w-16" />}
```

Nit, we do not need to use merge utility for icons in this case.

Reply:
Thanks for the tip! I've changed that as per your suggestion for all four icons.